shared/chat/conversation/info-panel/notifications/index.js

```diff
@@ -1,13 +1,31 @@
 // @flow
 import * as React from 'react'
-import {Box, Checkbox, Icon, RadioButton, Text} from '../../../../common-adapters'
+import {Box, Checkbox, Icon, RadioButton, ProgressIndicator, Text} from '../../../../common-adapters'
 import {globalColors, globalMargins, globalStyles, isMobile} from '../../../../styles'
+import {type NotificationSaveState} from '../../../../constants/types/chat'
 import type {Props} from '.'
 
+const SaveStateControl = (saveState: NotificationSaveState) => {
+  switch (saveState) {
+    case 'unsaved':
+      return null
+    case 'saving':
+      return <ProgressIndicator style={{alignSelf: 'center', width: globalMargins.medium}} />
+    case 'saved':
+      return [
+        <Icon key="0" type="iconfont-check" style={{color: globalColors.green}} />,
```

Comment:
Doesn't match the design exactly:

- The "Saved" message is centered. Probably better anyway, since the progress indicator barely shows up, and so it would look weird off-center.
- Baseline of the check mark isn't quite aligned with the text.

@cecileboucheron if you know any quick fixes

Reply:
Talked with @cecileboucheron, she says that the "Saved" message is supposed to be centered, the baseline is fine, but to set a height on the box so that it doesn't push anything down.

Reply:
Done.